.github/workflows/weekly_blog_and_website_report.yml

```diff
@@ -0,0 +1,53 @@
+name: Weekly blog and website report
+
+on:
+  schedule:
+    - cron: '0 4 * * WED'  # Run at 04:00 UTC every Wednesday
+  workflow_dispatch:
+
```

Comment:
This may make it easier to test, at the cost of slightly more Zulip spam:
```suggestion
  push:
```

Reply:
Ok, I'll remove `push` and merge.  I'd rather iterate a couple of times, than having a post for every push.